Gemfile.lock.release

```diff
@@ -916,7 +916,7 @@ GEM
       faraday (>= 0.9, < 2.0.0)
     psych (3.3.2)
     public_suffix (4.0.6)
-    puma (4.3.10)
```

Comment:
My only question is can you revisit the [config/puma.rb](https://github.com/ManageIQ/manageiq/blob/master/config/puma.rb#L20) and see if there were any changes in threads that might affect our current value and if there are any other settings added/changed that we might want to look at.  Note, the comment there is from 3.0.0 puma.

It's fine if it's a followup PR but it's worth revisiting those when we upgrade to see if anything needs to change.  For example, if the default is the same as our setting, we can probably drop `threads(5, 5)` from our configuration.

Reply:
@jrafanie `config/puma.rb` on `morphy` is the same as on `master`.
I did not find any changes that we would need.

Reply:
does this affect us setting min and max thread from configuration file? 

> We've removed the following public methods on Puma::Server: Puma::Server#min_threads, Puma::Server#max_threads. Instead, you can pass in configuration as an option to Puma::Server#new. This might make certain gems break (capybara for example).